Library/Homebrew/cmd/commands.rb

```diff
@@ -6,53 +6,63 @@
 
 module Homebrew
   module_function
-
   def commands
-    if ARGV.include? "--quiet"
-      cmds = internal_commands + external_commands
-      cmds += internal_developer_commands
-      cmds += HOMEBREW_INTERNAL_COMMAND_ALIASES.keys if ARGV.include? "--include-aliases"
-      puts Formatter.columns(cmds.sort)
-    else
-      # Find commands in Homebrew/cmd
-      puts "Built-in commands"
-      puts Formatter.columns(internal_commands)
-
-      # Find commands in Homebrew/dev-cmd
-      puts
-      puts "Built-in developer commands"
-      puts Formatter.columns(internal_developer_commands)
-
-      # Find commands in the path
-      unless (exts = external_commands).empty?
+    Homebrew::InstallCommand.commands
+  end
+
+  class InstallCommand < CommandOptions
```

Comment:
Hi @woodruffw! I didn't actually give much thought into naming it. I just used the name used by @bfontaine in the following comment: https://github.com/Homebrew/brew/pull/2420#issuecomment-295419433. Let me know if u want to change it